### ISSUE TITLE
Build of SonarAnalyzer.sln fails as it is not allowed to execute Set-ExecutionPolicy Unrestricted

### ISSUE THREAD

**Issue by Corniel** [SonarSource/sonar-dotnet#2223]
### Description
When building ~\sonaranalyzer-dotnet\SonarAnalyzer.sln I get the following error:
  MSB3073  The command "%WINDIR%\System32\WindowsPowerShell\v1.0\powershell.exe -command "if ((Get-ExecutionPolicy) -ne 'Unrestricted') {echo 'Powershell: error 0000: You have to run Set-ExecutionPolicy Unrestricted (In powershell32)'} " " exited with code -1.
 SonarAnalyzer.CSharp  ~\sonaranalyzer-dotnet\src\Rspec2Resx.targets	21	

### Repro steps
1. Get the latest version of the repro
2. Build the project

### Expected behavior
A compiled solution where I can run tests and add rules, not requiring some God-mode on a system.

### Actual behavior
Error

### Known workarounds
None, as my admin doesn't allow me to change this policy.

### Related information

* SonarC# Version 7.9.1.7622
* Visual Studio Version 2017 (15.9.4)

**Comment by Corniel** [comment]
I would love to here the expected time once this can be fixed. That would allow me to create some (compiling) pull requests. :)

**Comment by Evangelink** [comment]
Hi @Corniel,

I do understand the priority of this change so you could contribute but we are a bit short staffed at the moment so it's unlikely that we will be able to do something before next month :c

**Comment by Evangelink** [comment]
Hi @Corniel,

We did a small change which might help you, could you give it a go?

**Comment by Corniel** [comment]
I'll try it today. Keep you infromed.

**Comment by Corniel** [comment]
Builds like a charm. 4 tests fail:
```C#
ExecutingSqlQueries_CS_Disabled()
ExecutingSqlQueries_CS_NetCore()
ExecutingSqlQueries_VB_Disabled()
ExecutingSqlQueries_VB_NetCore()
```

Can not find the file specified.

**Comment by Evangelink** [comment]
Thanks for the feedback!

Could you let us know what is failing for those 4 tests?
